GoogleSignIn.podspec

```diff
@@ -13,7 +13,7 @@ The Google Sign-In SDK allows users to sign in with their Google account from th
     :tag => s.version.to_s
   }
   s.swift_version = '4.0'
-  ios_deployment_target = '10.0'
+  ios_deployment_target = '11.0'
```

Comment:
We may go to 12 in our fall major release since Xcode 15 drops iOS 11.

Reply:
Are y'all considering that for Firebase as a whole, or just App Check? Either way, is that something we can discuss wrt App Check? I'd like to do some research on GSI usage by iOS version. A quick look suggests not many clients are on < iOS 12, but I'd like to be sure.

Reply:
We're planning to move Firebase as a whole to 13 except for leaving Analytics and Crashlytics at 11. Let's discuss what to do about AppCheck.

Reply:
Following up here from an offline conversation. GSI is comfortable with an increase for the minimum iOS deployment target to 12.